samples/metrics_monitor/sample/cttmetrics_sample.cpp

```diff
@@ -64,9 +65,14 @@ int main(int argc, char *argv[])
         period_ms = atoi(argv[2]);
     }
 
+    if (3 < argc && NULL != argv[3])
```

Comment:
I think that's time to support --help option and print out help. Use getopt() standard function, please. I would like to see specification for device option format.